ExRam.Gremlinq.Providers.WebSocket/GremlinQueryEnvironmentExtensions.cs

```diff
@@ -228,9 +236,9 @@ public IGremlinQueryEnvironment Build()
 
         public static IGremlinQueryEnvironment UseWebSocket(
             this IGremlinQueryEnvironment environment,
-            Func<IWebSocketConfigurationBuilder, IGremlinQueryEnvironmentBuilder> builderAction)
+            Func<IWebSocketConfigurationBuilder, IGremlinQueryEnvironmentBuilder> builderAction, ConnectionPoolSettings connectionPoolSettings = null)
```

Comment:
Since there's SetConnectionPoolSettings on the builder, this parameter should not be there. It would defeat the purpose of the builder and introduce ambiguity.

Reply:
Seems I fixed it